User can't deploy two different DMN models in one deployment

**Describe the bug**

> Hi,
> 
> I’m having an issue trying to write a process test using Zeebe-process-test-extension in Camunda Cloud 8. (v.8.0.2)
> The following code runs fine if it’s just 1 bpmn and 1 dmn. But fails when i try adding second dmn:
> return client.newDeployResourceCommand()
> .addResourceFromClasspath(“test.bpmn”)
> .addResourceFile("~/dmn/test1.dmn")
> .addResourceFile("~/dmn/test2.dmn")
> .send()
> .join();
> 
> Here is the error: io.camunda.zeebe.db.ZeebeDbInconsistentException: Key DbLong{2251799813685250} in ColumnFamily DMN_DECISION_REQUIREMENTS already exists
> 
> I tried using addResourceFromClasspath and AddResourceFile - both methods result in the same error.

https://forum.camunda.io/t/fail-to-load-multiple-dmns-in-zeebe-process-test-extension-test-in-camunda-8/37483/4?u=zelldon


<!-- A clear and concise description of what the bug is. -->

**To Reproduce**


```
return client.newDeployResourceCommand()
.addResourceFromClasspath(“dummy_bpmn.bpmn”)
.addResourceFromClasspath(“dummy1.dmn”)
.addResourceFromClasspath(“dummy2.dmn”)
.send()
.join();
```

[dummy1.dmn.txt](https://github.com/camunda/zeebe/files/8650472/dummy1.dmn.txt)
[dummy2.dmn.txt](https://github.com/camunda/zeebe/files/8650473/dummy2.dmn.txt)

[dummy-bpmn.bpmn.txt](https://github.com/camunda/zeebe/files/8650474/dummy-bpmn.bpmn.txt)

<!--
Steps to reproduce the behavior

If possible add a minimal reproducer code sample
- when using the Java client: https://github.com/zeebe-io/zeebe-test-template-java

-->

**Expected behavior**

No error on deploying two different dmn models.

<!-- A clear and concise description of what you expected to happen. -->

**Log/Stacktrace**

<!-- If possible add the full stacktrace or Zeebe log which contains the issue. -->

<details><summary>Full Stacktrace</summary>
 <p>

```
Error:
io.camunda.zeebe.client.api.command.ClientStatusException: Command 'CREATE' rejected with code 'INVALID_ARGUMENT': Expected to deploy new resources, but encountered the following errors:
'dummy2.dmn': Key DbLong{2251799813685250} in ColumnFamily DMN_DECISION_REQUIREMENTS already exists

	at io.camunda.zeebe.client.impl.ZeebeClientFutureImpl.transformExecutionException(ZeebeClientFutureImpl.java:93)
	at io.camunda.zeebe.client.impl.ZeebeClientFutureImpl.join(ZeebeClientFutureImpl.java:50)
	at io.camunda.c8.test.DeterminationProcessTest.initDeployment(DeterminationProcessTest.java:48)
	at io.camunda.c8.test.DeterminationProcessTest.testDeployment(DeterminationProcessTest.java:53)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:725)
	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
	at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:149)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:140)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:84)
	at org.junit.jupiter.engine.execution.ExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(ExecutableInvoker.java:115)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.lambda$invoke$0(ExecutableInvoker.java:105)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:104)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:98)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$7(TestMethodTestDescriptor.java:214)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:210)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:135)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:66)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:107)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:88)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:67)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:52)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86)
	at org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86)
	at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:53)
	at com.intellij.junit5.JUnit5IdeaTestRunner.startRunnerWithArgs(JUnit5IdeaTestRunner.java:71)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
Caused by: java.util.concurrent.ExecutionException: io.grpc.StatusRuntimeException: INVALID_ARGUMENT: Command 'CREATE' rejected with code 'INVALID_ARGUMENT': Expected to deploy new resources, but encountered the following errors:
'dummy2.dmn': Key DbLong{2251799813685250} in ColumnFamily DMN_DECISION_REQUIREMENTS already exists
	at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:396)
	at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2073)
	at io.camunda.zeebe.client.impl.ZeebeClientFutureImpl.join(ZeebeClientFutureImpl.java:48)
	... 71 more
```

</p>
</details>

**Environment:**
- zeebe-process-test:8.0.1

Hi,

i think i now ran into probably the same bug now with ZeebeDeployment annotation:
@ZeebeDeployment(resources = {"classpath*:*.bpmn","classpath*:*.dmn"})

this fails with the same error as the above bud when there is more than 1 dmn found to be deployed.

Error:
Caused by: io.grpc.StatusRuntimeException: INVALID_ARGUMENT: Command 'CREATE' rejected with code 'INVALID_ARGUMENT': Expected to deploy new resources, but encountered the following errors:
'tooth.dmn': Key DbLong{2251799813685252} in ColumnFamily DMN_DECISION_REQUIREMENTS already exists

--- a/engine/src/main/java/io/camunda/zeebe/engine/processing/deployment/transform/DmnResourceTransformer.java
+++ b/engine/src/main/java/io/camunda/zeebe/engine/processing/deployment/transform/DmnResourceTransformer.java
@@ -8,6 +8,7 @@
 package io.camunda.zeebe.engine.processing.deployment.transform;
 
 import static io.camunda.zeebe.util.buffer.BufferUtil.wrapString;
+import static java.util.function.Predicate.not;
 
 import io.camunda.zeebe.dmn.DecisionEngine;
 import io.camunda.zeebe.dmn.DecisionEngineFactory;
@@ -68,8 +69,8 @@ public final class DmnResourceTransformer implements DeploymentResourceTransform
       return checkForDuplicateIds(resource, parsedDrg, deployment)
           .map(
               noDuplicates -> {
-                appendMetadataToDeploymentEvent(resource, parsedDrg, deployment);
-                writeRecords(deployment, resource);
+                final var drgKey = appendMetadataToDeploymentEvent(resource, parsedDrg, deployment);
+                writeRecords(deployment, resource, drgKey);
                 return null;
               });
 
@@ -148,7 +149,7 @@ public final class DmnResourceTransformer implements DeploymentResourceTransform
         .orElse("<?>");
   }
 
-  private void appendMetadataToDeploymentEvent(
+  private long appendMetadataToDeploymentEvent(
       final DeploymentResource resource,
       final ParsedDecisionRequirementsGraph parsedDrg,
       final DeploymentRecord deploymentEvent) {
@@ -223,6 +224,8 @@ public final class DmnResourceTransformer implements DeploymentResourceTransform
                               .setDecisionKey(newDecisionKey.getAsLong())
                               .setVersion(INITIAL_VERSION));
             });
+
+    return drgRecord.getDecisionRequirementsKey();
   }
 
   private boolean isDuplicate(
@@ -234,48 +237,44 @@ public final class DmnResourceTransformer implements DeploymentResourceTransform
         && drg.getChecksum().equals(checksum);
   }
 
-  private void writeRecords(final DeploymentRecord deployment, final DeploymentResource resource) {
-    final var metadataRecords =
-        deployment.decisionRequirementsMetadata().stream()
-            .filter(metadata -> metadata.getResourceName().equals(resource.getResourceName()))
-            .collect(Collectors.toList());
+  private void writeRecords(
+      final DeploymentRecord deployment,
+      final DeploymentResource resource,
+      final long decisionRequirementsKey) {
 
-    for (final DecisionRequirementsMetadataRecord drg : metadataRecords) {
-      if (!drg.isDuplicate()) {
-        stateWriter.appendFollowUpEvent(
-            drg.getDecisionRequirementsKey(),
-            DecisionRequirementsIntent.CREATED,
-            new DecisionRequirementsRecord()
-                .setDecisionRequirementsKey(drg.getDecisionRequirementsKey())
-                .setDecisionRequirementsId(drg.getDecisionRequirementsId())
-                .setDecisionRequirementsName(drg.getDecisionRequirementsName())
-                .setDecisionRequirementsVersion(drg.getDecisionRequirementsVersion())
-                .setNamespace(drg.getNamespace())
-                .setResourceName(drg.getResourceName())
-                .setChecksum(drg.getChecksumBuffer())
-                .setResource(resource.getResourceBuffer()));
-      }
-    }
+    deployment.decisionRequirementsMetadata().stream()
+        .filter(drg -> drg.getDecisionRequirementsKey() == decisionRequirementsKey)
+        .filter(not(DecisionRequirementsMetadataRecord::isDuplicate))
+        .findFirst()
+        .ifPresent(
+            drg ->
+                stateWriter.appendFollowUpEvent(
+                    drg.getDecisionRequirementsKey(),
+                    DecisionRequirementsIntent.CREATED,
+                    new DecisionRequirementsRecord()
+                        .setDecisionRequirementsKey(drg.getDecisionRequirementsKey())
+                        .setDecisionRequirementsId(drg.getDecisionRequirementsId())
+                        .setDecisionRequirementsName(drg.getDecisionRequirementsName())
+                        .setDecisionRequirementsVersion(drg.getDecisionRequirementsVersion())
+                        .setNamespace(drg.getNamespace())
+                        .setResourceName(drg.getResourceName())
+                        .setChecksum(drg.getChecksumBuffer())
+                        .setResource(resource.getResourceBuffer())));
 
-    final var decisionRequirementKeys =
-        metadataRecords.stream()
-            .map(DecisionRequirementsMetadataRecord::getDecisionRequirementsKey)
-            .collect(Collectors.toList());
-
-    for (final DecisionRecord decision : deployment.decisionsMetadata()) {
-      if (!decision.isDuplicate()
-          && decisionRequirementKeys.contains(decision.getDecisionRequirementsKey())) {
-        stateWriter.appendFollowUpEvent(
-            decision.getDecisionKey(),
-            DecisionIntent.CREATED,
-            new DecisionRecord()
-                .setDecisionKey(decision.getDecisionKey())
-                .setDecisionId(decision.getDecisionId())
-                .setDecisionName(decision.getDecisionName())
-                .setVersion(decision.getVersion())
-                .setDecisionRequirementsId(decision.getDecisionRequirementsId())
-                .setDecisionRequirementsKey(decision.getDecisionRequirementsKey()));
-      }
-    }
+    deployment.decisionsMetadata().stream()
+        .filter(decision -> decision.getDecisionRequirementsKey() == decisionRequirementsKey)
+        .filter(not(DecisionRecord::isDuplicate))
+        .forEach(
+            decision ->
+                stateWriter.appendFollowUpEvent(
+                    decision.getDecisionKey(),
+                    DecisionIntent.CREATED,
+                    new DecisionRecord()
+                        .setDecisionKey(decision.getDecisionKey())
+                        .setDecisionId(decision.getDecisionId())
+                        .setDecisionName(decision.getDecisionName())
+                        .setVersion(decision.getVersion())
+                        .setDecisionRequirementsId(decision.getDecisionRequirementsId())
+                        .setDecisionRequirementsKey(decision.getDecisionRequirementsKey())));
   }
 }

--- a/engine/src/main/java/io/camunda/zeebe/engine/processing/deployment/transform/DmnResourceTransformer.java
+++ b/engine/src/main/java/io/camunda/zeebe/engine/processing/deployment/transform/DmnResourceTransformer.java
@@ -235,8 +235,12 @@ public final class DmnResourceTransformer implements DeploymentResourceTransform
   }
 
   private void writeRecords(final DeploymentRecord deployment, final DeploymentResource resource) {
+    final var metadataRecords =
+        deployment.decisionRequirementsMetadata().stream()
+            .filter(metadata -> metadata.getResourceName().equals(resource.getResourceName()))
+            .collect(Collectors.toList());
 
-    for (final DecisionRequirementsMetadataRecord drg : deployment.decisionRequirementsMetadata()) {
+    for (final DecisionRequirementsMetadataRecord drg : metadataRecords) {
       if (!drg.isDuplicate()) {
         stateWriter.appendFollowUpEvent(
             drg.getDecisionRequirementsKey(),
@@ -253,8 +257,14 @@ public final class DmnResourceTransformer implements DeploymentResourceTransform
       }
     }
 
+    final var decisionRequirementKeys =
+        metadataRecords.stream()
+            .map(DecisionRequirementsMetadataRecord::getDecisionRequirementsKey)
+            .collect(Collectors.toList());
+
     for (final DecisionRecord decision : deployment.decisionsMetadata()) {
-      if (!decision.isDuplicate()) {
+      if (!decision.isDuplicate()
+          && decisionRequirementKeys.contains(decision.getDecisionRequirementsKey())) {
         stateWriter.appendFollowUpEvent(
             decision.getDecisionKey(),
             DecisionIntent.CREATED,

--- a/engine/src/test/java/io/camunda/zeebe/engine/processing/deployment/DeploymentDmnTest.java
+++ b/engine/src/test/java/io/camunda/zeebe/engine/processing/deployment/DeploymentDmnTest.java
@@ -36,6 +36,8 @@ public final class DeploymentDmnTest {
   private static final String DMN_DECISION_TABLE_V2 = "/dmn/decision-table_v2.dmn";
   private static final String DMN_DECISION_TABLE_RENAMED_DRG =
       "/dmn/decision-table-with-renamed-drg.dmn";
+  private static final String DMN_DECISION_TABLE_RENAMED_DRG_AND_DECISION =
+      "/dmn/decision-table-with-renamed-drg-and-decision.dmn";
 
   private static final String DMN_INVALID_EXPRESSION =
       "/dmn/decision-table-with-invalid-expression.dmn";
@@ -371,6 +373,55 @@ public final class DeploymentDmnTest {
             tuple("jedi_or_sith", 1, "force_users"),
             tuple("jedi_or_sith", 2, "star-wars"),
             tuple("jedi_or_sith", 3, "force_users"));
+  }
+
+  @Test
+  public void shouldDeployIfMultipleDrgHaveDifferentId() {
+    // when
+    final var deploymentEvent =
+        engine
+            .deployment()
+            .withXmlClasspathResource(DMN_DECISION_TABLE)
+            .withXmlClasspathResource(DMN_DECISION_TABLE_RENAMED_DRG_AND_DECISION)
+            .deploy();
+
+    // then
+    Assertions.assertThat(deploymentEvent)
+        .hasIntent(DeploymentIntent.CREATED)
+        .hasValueType(ValueType.DEPLOYMENT)
+        .hasRecordType(RecordType.EVENT);
+
+    assertThat(deploymentEvent.getValue().getDecisionRequirementsMetadata()).hasSize(2);
+
+    final var drgMetadata = deploymentEvent.getValue().getDecisionRequirementsMetadata().get(0);
+    Assertions.assertThat(drgMetadata).hasDecisionRequirementsId("force_users");
+    assertThat(drgMetadata.getDecisionRequirementsKey()).isPositive();
+    assertThat(drgMetadata.getChecksum())
+        .describedAs("Expect the MD5 checksum of the DMN resource")
+        .isEqualTo(getChecksum(DMN_DECISION_TABLE));
+
+    assertThat(deploymentEvent.getValue().getDecisionsMetadata()).hasSize(2);
+
+    final var decisionMetadata = deploymentEvent.getValue().getDecisionsMetadata().get(0);
+    Assertions.assertThat(decisionMetadata)
+        .hasDecisionId("jedi_or_sith")
+        .hasDecisionRequirementsKey(drgMetadata.getDecisionRequirementsKey());
+    assertThat(decisionMetadata.getDecisionKey()).isPositive();
+
+    final var drgMetadata2 = deploymentEvent.getValue().getDecisionRequirementsMetadata().get(1);
+    Assertions.assertThat(drgMetadata2).hasDecisionRequirementsId("star-wars");
+    assertThat(drgMetadata2.getDecisionRequirementsKey()).isPositive();
+    assertThat(drgMetadata2.getChecksum())
+        .describedAs("Expect the MD5 checksum of the DMN resource")
+        .isEqualTo(getChecksum(DMN_DECISION_TABLE_RENAMED_DRG_AND_DECISION));
+
+    assertThat(deploymentEvent.getValue().getDecisionsMetadata()).hasSize(2);
+
+    final var decisionMetadata2 = deploymentEvent.getValue().getDecisionsMetadata().get(1);
+    Assertions.assertThat(decisionMetadata2)
+        .hasDecisionId("sith_or_jedi")
+        .hasDecisionRequirementsKey(drgMetadata2.getDecisionRequirementsKey());
+    assertThat(decisionMetadata2.getDecisionKey()).isPositive();
   }
 
   @Test

--- a/engine/src/test/resources/dmn/decision-table-with-renamed-drg-and-decision.dmn
+++ b/engine/src/test/resources/dmn/decision-table-with-renamed-drg-and-decision.dmn
@@ -1,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<definitions xmlns="https://www.omg.org/spec/DMN/20191111/MODEL/" xmlns:dmndi="https://www.omg.org/spec/DMN/20191111/DMNDI/" xmlns:dc="http://www.omg.org/spec/DMN/20180521/DC/" xmlns:biodi="http://bpmn.io/schema/dmn/biodi/2.0" id="star-wars" name="Star Wars" namespace="http://camunda.org/schema/1.0/dmn" exporter="Camunda Modeler" exporterVersion="5.0.0">
+  <decision id="sith_or_jedi" name="Jedi or Sith">
+    <decisionTable id="DecisionTable_14n3bxx">
+      <input id="Input_1" label="Lightsaber color" biodi:width="192">
+        <inputExpression id="InputExpression_1" typeRef="string">
+          <text>lightsaberColor</text>
+        </inputExpression>
+      </input>
+      <output id="Output_1" label="Jedi or Sith" name="jedi_or_sith" typeRef="string" biodi:width="192">
+        <outputValues id="UnaryTests_0hj346a">
+          <text>"Jedi","Sith"</text>
+        </outputValues>
+      </output>
+      <rule id="DecisionRule_0zumznl">
+        <inputEntry id="UnaryTests_0leuxqi">
+          <text>"blue"</text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_0c9vpz8">
+          <text>"Jedi"</text>
+        </outputEntry>
+      </rule>
+      <rule id="DecisionRule_1utwb1e">
+        <inputEntry id="UnaryTests_1v3sd4m">
+          <text>"green"</text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_0tgh8k1">
+          <text>"Jedi"</text>
+        </outputEntry>
+      </rule>
+      <rule id="DecisionRule_1bwgcym">
+        <inputEntry id="UnaryTests_0n1ewm3">
+          <text>"red"</text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_19xnlkw">
+          <text>"Sith"</text>
+        </outputEntry>
+      </rule>
+    </decisionTable>
+  </decision>
+  <dmndi:DMNDI>
+    <dmndi:DMNDiagram>
+      <dmndi:DMNShape dmnElementRef="sith_or_jedi">
+        <dc:Bounds height="80" width="180" x="160" y="100" />
+      </dmndi:DMNShape>
+    </dmndi:DMNDiagram>
+  </dmndi:DMNDI>
+</definitions>

